packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -77,7 +77,7 @@ class Checkbox extends SlotLabelMixin(
           -webkit-tap-highlight-color: transparent;
         }
 
-        [part='container'] {
```

Comment:
Just nitpicking about the commit message - what do you think about adding refactor! here? This looks like a breaking change from the outside.. because I've seen people using the part=container because it's a public api that now gets removed (to change e.g. the flex alignment).

Reply:
Good point. Updated to use `refactor!` - technically, this part did not exist in previous Vaadin versions.

Reply:
That particular Cookbook recipe looks to be using some private JS APIs as well, so I think it doesn’t make much difference if it also has to use some private theming API as well 🤷